frontend/public/components/RBAC/bindings.jsx

```diff
@@ -639,7 +643,7 @@ export const CreateRoleBinding = ({ match: { params }, location }) => {
   });
   const fixed = {
     kind: params.ns || roleKind === 'Role' || !clusterAllowed ? 'RoleBinding' : undefined,
-    metadata: { namespace: params.ns },
```

Comment:
I think this will break setting the namespace when creating a role binding from a namespaced role. It would be better to generate the namespaced URL like in this screenshot:

<img width="1023" alt="Create Role Binding · OKD 2019-10-31 20-23-16" src="https://user-images.githubusercontent.com/1167259/67994583-53fec480-fc1c-11e9-8f58-038326780272.png">